Examples/restcomm-olympus-nopod/restcomm-olympus/SigninTableViewController.m

```diff
@@ -118,36 +115,45 @@ - (RCDevice *)registerRCDevice{
     return [appDelegate registerRCDevice];
 }
 
-- (void)registerForPushWithAccount:(NSString *)account withRCDevice:(RCDevice *)rcDevice password:(NSString *)password andEmail:(NSString *)email{
-    //get token from user defaults
-    NSUserDefaults* appDefaults = [NSUserDefaults standardUserDefaults];
-    NSString *deviceToken = [appDefaults objectForKey:@"deviceToken"];
-    if (deviceToken && rcDevice){
-        //get certificate strings
-        NSString *pushCertificatesPathPublic = [[NSBundle mainBundle] pathForResource:@"certificate_key_push" ofType:@"pem"];
-        NSString *pushCertificatesPathPrivate = [[NSBundle mainBundle] pathForResource:@"rsa_private_key_push" ofType:@"pem"];
+- (void)registerForPush{
+    //before setting the push token, we need to check is push enabled (server can handle it)
+    if ([Utils isServerEnabledForPushNotifications]){
+        NSLog(@"Start registering for push on server");
+        NSString *deviceToken = [Utils pushToken];
+        NSString *pushAccount = [Utils pushAccount];
         
-      
-
-        NSMutableDictionary *dic = [[NSMutableDictionary alloc] initWithObjectsAndKeys:
-                                    @"Olympus", @"friendly-name",
-                                    account, @"username",
-                                    password, @"password",
-                                    email, @"rescomm-account-email",
-                                    deviceToken, @"token",
-                                    pushCertificatesPathPublic, @"push-certificate-public-path",
-                                    pushCertificatesPathPrivate, @"push-certificate-private-path",
-                                    [NSNumber numberWithBool:YES], @"is-sandbox", nil];
-        
-        [rcDevice registerPushToken:dic delegate:self];
-        
-    } else {
-        NSLog(@"Device Voip push token not found, or RCDevice not initialized");
+        if (deviceToken && pushAccount){
+            AppDelegate *appDelegate = ((AppDelegate *)[UIApplication sharedApplication].delegate);
+            RCDevice *rcDevice = [appDelegate registerRCDevice];
+            if (rcDevice){
+                //get certificate strings
+                NSString *pushCertificatesPathPublic = [[NSBundle mainBundle] pathForResource:@"certificate_key_push" ofType:@"pem"];
+                NSString *pushCertificatesPathPrivate = [[NSBundle mainBundle] pathForResource:@"rsa_private_key_push" ofType:@"pem"];
```

Comment:
We agree not to encrypt it. The content of the files are pem's strings because server is requires them in those formats (check registerPushToken NOTE). Even if someone take those certificates, they will not be able to send push because they need to have right push token of the device to do it. 
Also, if someones want to use our certificates in theirs app, apple will not allow it because certificate belongs to our bundle id.

Reply:
Ok I remember now. Let's leave it as is for now, but even if its unusable, ultimately we shouldn't allow access to any private key. Can you add a bullet about that in the design doc on current things missing/todo?